### DISCUSSION TITLE
Feature request: Some special footer link open to oss version

### DISCUSSION THREAD

**Discussion by iceking2nd** [grafana/grafana#70226] [General]
According to the requirements of the Chinese government, any page that can be accessed by the Internet must add the website record number of the Ministry of Industry and Information Technology and the website record number of the Ministry of Public Security. Can you consider opening these two special footer link functions to the OSS version of Grafana?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82646. Feel free to continue the discussion around this there. Thank you!
